<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChangeScreen</class>
 <widget class="QDialog" name="PasswordChangeScreen">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 13, 13);
</string>
  </property>
  <widget class="QPushButton" name="potwierdz">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>434</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color: white;

font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>POTWIERDŹ</string>
   </property>
  </widget>
  <widget class="QLabel" name="forma">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>200</y>
     <width>300</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>NOWE HASŁO:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kupujesz">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>100</y>
     <width>300</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);
border-style: none;</string>
   </property>
   <property name="text">
    <string>STARE HASŁO:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>554</y>
     <width>400</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color: white;

font: 75 30pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ANULUJ</string>
   </property>
  </widget>
  <widget class="QLabel" name="forma_2">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>300</y>
     <width>300</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: none;
font: 75 19pt &quot;Tahoma&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>POWTÓRZ NOWE HASŁO:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPassword">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>100</y>
     <width>385</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: none;
border-width:4px;
border-radius:30px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPassword">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>200</y>
     <width>385</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: none;
border-width:4px;
border-radius:30px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPassword2">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>300</y>
     <width>385</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
font: 75 30pt &quot;Tahoma&quot;;

border-style: none;
border-width:4px;
border-radius:30px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>84</y>
     <width>700</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);
border-style: solid;
border-width:4px;
border-radius:30px;
border-color:  rgb(218, 0, 0);;</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>171</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>862</x>
     <y>-10</y>
     <width>171</width>
     <height>791</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>690</y>
     <width>721</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>-10</y>
     <width>721</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 13, 13);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton</zorder>
  <zorder>potwierdz</zorder>
  <zorder>forma</zorder>
  <zorder>kupujesz</zorder>
  <zorder>close</zorder>
  <zorder>forma_2</zorder>
  <zorder>oldPassword</zorder>
  <zorder>newPassword</zorder>
  <zorder>newPassword2</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
